webapp/views.py

```diff
@@ -75,7 +75,7 @@ def get_cve(cve_id, **kwargs):
 @use_kwargs(CVEsParameters, location="query")
 def get_cves(**kwargs):
     query = kwargs.get("q", "").strip()
-    priority = kwargs.get("priority")
+    priorities = kwargs.get("priority")
```

Comment:
I'd pass all values with named arguments e.g. `get_cves(priorities...` it'd be safer but I understand it may need major refactoring

Reply:
Major refactoring indeed 😅